traja/models/train.py

```diff
@@ -132,7 +132,7 @@ def __str__(self):
         return f"Training model type {self.model_type}"
 
     def fit(
-        self, dataloaders, model_save_path=None, training_mode="forecasting", epochs=50
+            self, dataloaders, model_save_path=None, training_mode="forecasting", epochs=50, validate_every=3, test_every=10
```

Comment:
The new arguments should be described in the function docstring. Also, we should allow `validate_every=None` to disable training-time validation altogether.

Reply:
Done.